### PR TITLE
Airport data

### PR DESCRIPTION
**Pull request by MariusHahn**

### CODE REVIEW

File: backend/src/orm/airports.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from geoalchemy2 import Geography
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import func
+# from typing import NamedTuple
+
+Base = declarative_base()
+
+class Report(Base):
```
Comment (MariusHahn) [inline backend/src/orm/airports.py]: should have the same name as your table and

File: backend/src/orm/airports.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    __tablename__ = 'airports'
```
Comment (MariusHahn) [inline backend/src/orm/airports.py]: It is common to use singular for table names. I would change it to 'airport'

File: backend/src/orm/airports.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    airport_id = Column(Integer, primary_key=True)
```
Comment (MariusHahn) [inline backend/src/orm/airports.py]: As you are in airport its clear that it belongs to airport. Why not just taking id as name

File: backend/src/orm/airports.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            cls.airport_id,
+            cls.iata_code,
+            cls.name,
+            func.st_asgeojson(cls.coordinates).label("geojson")
+            cls.elevation_ft,
+            cls.country_code,
+            cls.type_size,
+            )
+
+    # @staticmethod
```
Comment (MariusHahn) [inline backend/src/orm/airports.py]: Is it needed? Through it away or make it callable

File: database/airports/airports_to_database.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+if (process.argv.length === 2 || process.argv[2] === '0') {
+	create_table_string = 'DROP TABLE IF EXISTS airports;'
+		+ 'CREATE TABLE airports ('
+  		+ 'airport_id int NOT NULL PRIMARY KEY,'
+  		+ 'iata_code varchar(10),'
+  		//+ 'icao varchar(4) NOT NULL,'
+  		+ 'name varchar(100) NOT NULL,'
+  		//+ 'name_de varchar(250),'
+  		// + 'length_ft smallint,'
```
Comment (MariusHahn) [inline database/airports/airports_to_database.js]: Same here. Is it needed?

File: database/airports/airports_to_database.js [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+  		+ 'type_size varchar(50),'
+  		+ 'name varchar(200) NOT NULL,'
+  		// + 'latitude decimal NOT NULL,'
+  		// + 'longitude decimal NOT NULL,'
+  		+ 'elevation_ft int,'
+  		//+ 'continent varchar(200),'
+  		+ 'country_code varchar(5),'
+  		//+ 'iso_region varchar(200),'
+  		//+ 'municipality varchar(200),'
+  		//+ 'scheduled_service varchar(200),'
```
Comment (MariusHahn) [inline database/airports/airports_to_database.js]: is it needed?

File: backend/src/orm/airports.py (hunk 4 above)
Comment (MariusHahn) [reply backend/src/orm/airports.py]: Actuall it can be useful if you want to have some human readable coordinates

**Review by robin-gerling — APPROVED** [review]

**Pull request merged**